src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -18,25 +23,68 @@
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " alice bob charlie";
+    public static final String WARNING_INVALID_CONTEXT = "This command can only be used"
+            + "while viewing the list of activities/contacts.";
+    private final String searchTerm;
+    private final String[] keywords;
 
-    private final NameContainsKeywordsPredicate predicate;
-
-    public FindCommand(NameContainsKeywordsPredicate predicate) {
-        this.predicate = predicate;
+    public FindCommand(String[] keywords, String searchTerm) {
+        this.keywords = keywords;
+        this.searchTerm = searchTerm;
+        System.out.print("Keywords: ");
```

Comment:
If this is intended for debugging purposes, we should use a `Logger` instead of printing directly using `System.out.println`, and set it to print at a higher debug level instead.

Use `grep -r Logger` to find any of the existing files that already use the `Logger` and `LogsCentre` already, then just replicate their style.

Also, the debug output for the keywords need to be cleaned up somewhat (e.g. odd trailing comma).

![image](https://user-images.githubusercontent.com/44989315/68181268-abf94a80-ffd1-11e9-8e5c-741ef6e5624e.png)

Reply:
Whoops, this is a leftover from my debugging. Will remove this entirely

Reply:
Removed unused logging code